src/site/content/en/blog/user-agent-client-hints/index.md

```diff
@@ -337,7 +343,7 @@ for that site until the browser is closed.
 ⬆️ _Subsequent requests_
 
 ```text
-Sec-CH-UA-Full-Version: "84.0.4143.2"
+Sec-CH-UA-Full-Version: "93.0.4577.63"
```

Comment:
Maybe update the example just below to use `Sec-CH-UA-Bitness` just to avoid any confusion with platform.

Reply:
Done.